test/extensions/filters/http/common/fuzz/filter_fuzz_test.cc

```diff
@@ -11,6 +11,12 @@ namespace Extensions {
 namespace HttpFilters {
 
 DEFINE_PROTO_FUZZER(const test::extensions::filters::http::FilterFuzzTestCase& input) {
+
+  static const absl::flat_hash_set<absl::string_view> exclusion_list = {
+      "envoy.ext_proc", "envoy.filters.http.alternate_protocols_cache",
+      "envoy.filters.http.composite", "envoy.filters.http.ext_proc",
+      "envoy.filters.http.ext_authz"};
```

Comment:
can you document in https://github.com/envoyproxy/envoy/issues/20737 why ext_authz doesn't work with this fuzzer since we're adding it here? Thanks

Reply:
Added a comment ref'ing both issues.